docs/using.rst

```diff
@@ -1,45 +1,109 @@
-==============================
-Using the Let's Encrypt client
-==============================
+==========
+User Guide
+==========
 
+.. contents:: Table of Contents
+   :local:
 
-Getting the code
-================
+.. _installation:
 
-Please `install Git`_ and run the following commands:
+Installation
+============
+
+Unless you have a very specific requirements, we kindly ask you to use
+the letsencrypt-auto_ method described below. It's the fastest, the
```

Comment:
I don't actually agree with this comment. I think narrative introduction is necessary for reader to digest following material properly. However, I don't want to hold off this PR anymore, so I move this paragraph to a section at the bottom: "Comparison of different methods".